packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -17,7 +17,9 @@ import { ReactElement, Ref, forwardRef } from 'react';
 import { css } from '@emotion/core';
 
 import styled, { StyleProps } from '../../styles/styled';
-import { ButtonProps } from '../Button/Button';
+import Button, { ButtonProps } from '../Button';
+
+type Action = Omit<ButtonProps, 'variant'>;
 
 export interface ButtonGroupProps {
```

Comment:
Let's deprecate the `children` prop and encourage devs to use the `action` prop instead.